### PR TITLE
fix(tooltip): add forward ref

### PR DESCRIPTION
**Pull request by David-Cohen35**

### CODE REVIEW

File: src/components/Tooltip/index.tsx [hunk 1]
```diff
@@ -16,31 +16,36 @@ export interface TooltipProps {
...
+    { trigger, message, position = 'bottom', maxWidth = 240 }: TooltipProps,
+    forwardedRef
+  ) => {
+    const refContent = forwardedRef || useRef<HTMLDivElement>(null);
+
+    return (
+      <RadixTooltip.Provider>
+        <RadixTooltip.Root delayDuration={350}>
+          <RadixTooltip.Trigger asChild>
+            <span ref={refContent}>{trigger}</span>
```
Comment (ivanhdez28) [inline src/components/Tooltip/index.tsx:30]: i kind of feel that we agreed on forwarding the ref to the trigger and removing the span, didn't we?
i could be wrong.

Comment (David-Cohen35) [reply src/components/Tooltip/index.tsx:30]: RadixTooltip.Trigger doesn't allow a ref prop on it

Comment (ivanhdez28) [reply src/components/Tooltip/index.tsx:30]: gotcha, then I guess passing it to the `<span>` and using that `asChild` makes sense

**Review by ivanhdez28 — APPROVED** [review]

File: src/components/Tooltip/index.tsx (hunk 1 above)
Comment (David-Cohen35) [reply src/components/Tooltip/index.tsx:30]: yeah idk if there's any other way to do it

**Pull request merged**